docs/en/docs/kant/user-guide/device/create-device.md

```diff
@@ -1,47 +1,59 @@
-# Creating End Device
+---
+MTPE: FanLin
+Date: 2024-01-05
+---
 
-End device can be connected to edge nodes and support access via the Modbus protocol.
-Once an end device is connected, it can be managed centrally in the cloud management plane.
+# Create End Device
+
+End device can be connected to edge nodes and support access via the Modbus protocol. Once an end device is connected, it can be managed centrally in the cloud management plane.
```

Comment:
```suggestion
End device can be connected to edge nodes and support access via the Modbus protocol.
Once an end device is connected, it can be managed centrally in the cloud management plane.
```
Tweaking long lines can be beneficial to reviewers if any changes occur next time.